### PR TITLE
Added Linux Deepin icon

### PR DESCRIPTION
**Pull request by JoaoPedroMoraes**
**Issue:** #1160 

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
From the official SVG I was able to create a monocolor version. https://www.deepin.org/en/

### CODE REVIEW

**Comment by birjolaxew** [comment]
Can you elaborate on which official SVG you are talking about? I haven't been able to find any on the deepin website.

Also they seem to use lowercase "deepin" when referring to the operating system, with "Deepin" being used for any related name (see e.g. [deepin Wiki](https://wiki.deepin.org/wiki/Native_install): "Visit the Deepin Community Download Page to download the latest deepin ISO/Live System.")

**Comment by JoaoPedroMoraes** [comment]
> Você pode explicar em qual SVG oficial você está falando? Eu não consegui encontrar nenhum no site da deepin.
> 
> Além disso, eles parecem usar minúsculas "deepin" quando se referem ao sistema operacional, com "Deepin" sendo usado para qualquer nome relacionado (veja, por exemplo, [deepin Wiki](https://wiki.deepin.org/wiki/Native_install) : "Visite a página de download da Deepin Community para baixar o mais recente sistema Deepin ISO / Live". )

The svg I took from the distro itself (is the one I use) and only adapted to the standard. This is the symbol of the distro, without the tiny deepin as you have observed.

**Comment by JoaoPedroMoraes** [comment]
If you want to know more information about deepin https://www.deepin.org/en/aboutus/
and where can we find the logo on svg https://commons.wikimedia.org/wiki/File:Deepin_logo.svg

**Comment by ericcornelissen** [comment]
> If you want to know more information about deepin https://www.deepin.org/en/aboutus/
and where can we find the logo on svg https://commons.wikimedia.org/wiki/File:Deepin_logo.svg

In that case, can you update the source in `simple-icons.json` to the Wikimedia URL?

**Comment by JoaoPedroMoraes** [comment]
> > Se você quiser saber mais informações sobre deepin https://www.deepin.org/en/aboutus/ 
> > e onde podemos encontrar o logotipo em svg https://commons.wikimedia.org/wiki/File:Deepin_logo.svg
> 
> Nesse caso, você pode atualizar a fonte `simple-icons.json`para o URL da Wikimedia?

update made

**Comment by JoaoPedroMoraes** [comment]
Do I need to do anything else in this pr?

**Comment by ericcornelissen** [comment]
> Do I need to do anything else in this pr?

I'm personally not sure about the name. I agree with @birjolaxew that it seems the distro name is 'deepin' (with a lowercase 'd', see also [Wikipedia](https://en.wikipedia.org/wiki/Deepin): "Deepin (stylized as deepin; [...]") but everywhere else it is Deepin (with a uppercase 'D', e.g. Deepin Technologies). So the question really is, do we add the icons as belonging to the distro or to the organization, as far as I can tell they both use the same logo... 🤔 

Either way, the name will have to be updated... Do you have something to say about this @JoaoPedroMoraes? Any other opinions @birjolaxew and @davidklebanoff?

**Comment by davidklebanoff** [comment]
I think "deepin", lowercase, is the most appropriate, since it's often stylized as such when referencing the distro, which I believe is more popular than the Deepin organization itself.

**Comment by JoaoPedroMoraes** [comment]
I would like to merge pr, what do I still need to do?

**Comment by davidklebanoff** [comment]
Sorry @JoaoPedroMoraes, it looks like this PR got put on pause as we were trying to get consensus on the icon name. My vote is "deepin", for reasons mentioned above. What are your votes @ericcornelissen, @birjolaxew?

**Comment by birjolaxew** [comment]
@davidklebanoff I agree with you - when talking about the distro, it seems that "deepin" is what they use.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -610,6 +610,11 @@
             "hex": "A81D33",
             "source": "https://www.debian.org/logos"
         },
+        {
+            "title": "Deepin",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Fully agree @davidklebanoff, if this is changed this PR is good to go @JoaoPedroMoraes :wink: 

```suggestion
            "title": "deepin",
```

**Comment by JoaoPedroMoraes** [comment]
I do not know why it is not passing the test, could you clarify the error?

**Comment by birjolaxew** [comment]
@JoaoPedroMoraes It's because the `"title": "deepin"` in the JSON and the `<title>Deepin icon</title>` don't match - to fix it you'll just need to update it to `<title>deepin icon</title>` :)

**Comment by JoaoPedroMoraes** [comment]
solved problem

**Comment by birjolaxew** [comment]
:+1: Looks good! Thanks for your work on this, especially fixing the (minor) issues we had. Your work is much appreciated!

**Pull request merged**